modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/PrepareMasterDatabaseForRelease.pm

```diff
@@ -0,0 +1,282 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Parts::PrepareMasterDatabaseForRelease
+
+=head1 DESCRIPTION
+
+    This is a partial PipeConfig for most part of the PrepareMasterDatabaseForRelease
+    pipeline. This will update the NCBI taxonomy, add/update all species to master
+    database, update master database's metadata, and update collections and mlss.
+    Finally, it will run the healthchecks and perform a backup of the updated master
+    database.
+
+=head1 AUTHORSHIP
+
+Ensembl Team. Individual contributions can be found in the GIT log.
+
+=head1 APPENDIX
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::Parts::PrepareMasterDatabaseForRelease;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Version 2.4;
+
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+sub pipeline_analyses_prep_master_db_for_release {
+    my ($self) = @_;
+    return [
+        {   -logic_name => 'patch_master_db',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PrepareMaster::PatchMasterDB',
+            -parameters => {
+                'schema_file'  => $self->o('schema_file'),
+                'patch_dir'    => $self->o('patch_dir'),
+                'prev_release' => '#expr( #release# - 1 )expr#',
+                'patch_names'  => '#patch_dir#/patch_#prev_release#_#release#_*.sql',
+            },
+            -flow_into  => ['load_ncbi_node'],
```

Comment:
Sorry, I thought since it was changed into a Parts module and moved into a new path, it wouldn't track anymore the origin of the contents. I'll keep it in mind in the future.